app/Http/Controllers/registrationController.php

```diff
@@ -3,11 +3,98 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Models\Users;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Redis;
+use App\Models\tran;
+use Illuminate\Support\Facades\Redirect;
 
 class registrationController extends Controller
 {
-    public function index()
+    public function registerShow()
     {
         return view('form');
     }
+    public function loginShow()
+    {
+        return view('login');
+    }
+    public function register()
+    {
+        return redirect('welcome');
+        
+    }
+    public function view(){
+        $people = Users::latest('updated_at')->get();
+        $data = compact('people');
+        return view('customer-view')->with($data);
```

Comment:
```suggestion
        return view('customer_view')->with($data);
```
Use snake_case to name your blade files,
Ref: https://stackoverflow.com/a/61811356

Reply:
similar changes throughout